Source/SvgExtentions.cs

```diff
@@ -21,12 +21,12 @@ public static void SetRectangle(this SvgRectangle r, RectangleF bounds)
             r.Width = bounds.Width;
             r.Height = bounds.Height;
         }
-
+#if !NO_SDC
         public static RectangleF GetRectangle(this SvgRectangle r)
         {
             return new RectangleF(r.X, r.Y, r.Width, r.Height);
         }
-
+#endif

```

Comment:
This should not be factored out?

Reply:
I would leave that when we split into separate project, there are more such issues to solve

Reply:
Ok, I think we can merge this, thanks!